src/ddspdrum/module.py

```diff
@@ -353,10 +350,14 @@ def __init__(
         sustain_duration: float,
         pitch_adsr: ADSR = ADSR(),
         amp_adsr: ADSR = ADSR(),
-        vco: VCO = VCO(),
+        vco: Union[VCO, List[VCO]] = VCO(),
         vca: VCA = VCA(),
+        noise_ratio: float = 0
     ):
-        super().__init__(modules=[pitch_adsr, amp_adsr, vco, vca])
+        if type(vco) is not list:
+            vco = [vco]
+
```

Comment:
This feels messy. Not sure if this is the best way to do this (i.e., accept arbitrary number of oscillators).

Reply:
Force it to be a Tuple (not List) instead of Union. Each should have a weight between 0 and 1. Don't enforce that they sum to one.

I still think at some point soon we'll want to make a decision upon a default specific synth structure, i.e. number of oscillators, since learning through a variable structure is hard. We could keep the generic var-osc thing, but I think the end-user API we should just pick a sensible default.

Reply:
Ok then, shall we go with two oscillators?

Reply:
Sure